pages/linux/fixfiles.md

```diff
@@ -0,0 +1,36 @@
+# fixfiles
+
+> Fix file SELinux security contexts.
+> More information: <https://manned.org/fixfiles>.
+
+- If specified with onboot, this fixfiles will record the current date in the `/.autorelabel` file, so that it can be used later to speed up labeling. If used with restore, the restore will only affect files that were modified today:
+
+`fixfiles -B`
+
+- Force reset of context to match `file_context` for customizable files:
+
+`fixfiles -F`
+
+- Clear `/tmp` directory with out prompt for removal:
+
+`fixfiles -f`
+
+- Use the rpm database to discover all files within the specified packages and restore the file contexts:
+
+`fixfiles -R rpmpackagename[,rpmpackagename...]`
+
+- Run a diff on the `PREVIOUS_FILECONTEXT` file to the currently installed one, and restore the context of all affected files:
+
+`fixfiles -C PREVIOUS_FILECONTEXT`
+
+- Only act on files created after the specified date.  Date must be specified in "YYYY-MM-DD HH:MM" format.  Date field will be passed to find --newermt command:
+
+`fixfiles -N {{time}}`
```

Comment:
```suggestion
- Only act on files created after a specific date which will be passed to find `--newermt` command:

`fixfiles -N {{YYYY-MM-DD HH:MM}}`
```

Almost the same issue as above.
Optional fix: option name in description can be surrounded with backticks.
Optional fix: placeholder content can be changed to show expected time format.